salt/modules/win_wua.py

```diff
@@ -110,7 +110,7 @@ def available(software=True,
             Include software updates in the results (default is True)
 
         drivers (bool):
-            Include driver updates in the results (default is False)
+            Include driver updates in the results (default is True)
```

Comment:
It looks like this was always wrong, the result of a copypasta when this function was added. I've opened https://github.com/saltstack/salt/pull/43742 to fix this in the 2017.7 branch as well.